### PR TITLE
Feat/saswwn

### PR DESCRIPTION
**Pull request by jskazinski**
The main purpose of this request is to update the CSI Driver's usage of csi-lib-sas to the new API changes.

Release Notes:
- Upgrade to github.com/Seagate/csi-lib-sas v1.0.0
- Change to use only Volume WWN for SAS Connector
- Changes to saslib.Attach to pass pointer to Connector, since values are updated
- The field connector.DevicePath was renamed connector.TargetDevice
- Connector.TargetDevice is a SCSI drive or a multipath device based on connector.Multipath true|false

### CODE REVIEW

**Review by David-T-White — APPROVED** [review]
looks good!

File: pkg/storage/sasNode.go [hunk 1]
```diff
@@ -71,13 +71,10 @@ func (sas *sasStorage) NodePublishVolume(ctx context.Context, req *csi.NodePubli
...
 	klog.V(1).Infof("[START] publish volume (%s) wwn (%s) target (%s) lun (%s)", volumeName, wwn, req.GetTargetPath(), lun)
 
-	wwns := req.GetVolumeContext()[common.WWNs]
-	klog.Infof("SAS WWNS: %s", wwns)
-
 	// Initiate SAS attachment
 	klog.Info("initiating SAS connection...")
-	connector := saslib.Connector{Lun: lun, TargetWWNs: []string{wwns}}
-	path, err := saslib.Attach(ctx, connector, &saslib.OSioHandler{})
+	connector := saslib.Connector{TargetWWN: wwn}
```
Comment (seagate-chris) [inline pkg/storage/sasNode.go:76]: This should be VolumeWWN since your latest changes?

**Review by seagate-chris — COMMENTED** [review]
I think this PR is out of date w/r/t the latest csi-lib-sas changes?

File: pkg/storage/sasNode.go (hunk 1 above)
Comment (jskazinski) [reply pkg/storage/sasNode.go]: Yes, I was waiting for csi-lib-sas changes, sorry about that. I have now committed the changes that match the latest sas library and I have also update go.mod/go.sum accordingly to the latest release tag.

**Review by seagate-chris — APPROVED** [review]

**Pull request merged**